src/test/java/com/teragrep/pth_06/planner/bloomfilter/TableFiltersTest.java

```diff
@@ -147,13 +150,35 @@ public void testInsertFiltersIntoCategoryTable() {
                 .getTables()
                 .get(0);
         DataAccessException exception = Assertions
-                .assertThrows(DataAccessException.class, () -> new TableFilters(ctx, table, 0L, "192.168.1.1").insertFiltersIntoCategoryTable());
-        Assertions.assertTrue(exception.getMessage().contains("term_0_target\" (term_id, type_id, \"filter\")"));
+                .assertThrows(DataAccessException.class, () -> new TableFilters(ctx, table, 0L, "192.168.1.1").asBatch().execute());
+        Assertions
+                .assertTrue(
+                        exception
+                                .getMessage()
+                                .contains("insert into \"term_0_target\" (\"term_id\", \"type_id\", \"filter\") values")
+                );
     }
 
     @Test
-    public void testInsertFiltersWithoutPatternMatch() {
-        fillTargetTable();
+    public void testInsertFiltersIntoCategoryTableRegexExtract() {
```

Comment:
please rename exception throwing test so that it describes why exception is expected

Reply:
updated test to check for values inserted into temp table and added separate exception test when temp table is not in database.